src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceWorkflow.kt

```diff
@@ -0,0 +1,76 @@
+package com.cognifide.gradle.aem.instance.tasks
+
+import com.cognifide.gradle.aem.common.instance.Instance
+import com.cognifide.gradle.aem.common.instance.names
+import com.cognifide.gradle.aem.common.instance.service.repository.Node
+import com.cognifide.gradle.aem.common.instance.service.repository.ResourceType
+import com.cognifide.gradle.aem.common.instance.service.workflow.WorkflowException
+import com.cognifide.gradle.aem.common.instance.service.workflow.WorkflowScheduler
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+import com.cognifide.gradle.aem.common.tasks.Instance as InstanceTask
+
+open class InstanceWorkflow : InstanceTask() {
+
+    private val notifier = common.notifier
+
+    @Internal
+    val model = aem.obj.string {
+        common.prop.string("instance.workflow")?.let { set(it) }
+    }
+
+    @Internal
+    val path = aem.obj.string {
+        convention(Node.DAM_PATH)
+        common.prop.string("instance.workflow.path")?.let { set(it) }
+    }
+
+    @Internal
+    val resourceType = aem.obj.string {
+        convention(ResourceType.ASSET.value)
+        common.prop.string("instance.workflow.resourceType")?.let { set(it) }
+    }
+
+    @TaskAction
+    fun run() {
+        instanceManager.examine(anyInstances)
+
+        if (!model.isPresent) throw WorkflowException("Workflow model is not specified, please specify it via 'instance.workflow' property")
```

Comment:
`Workflow model is not defined! Specify it by property 'instance.workflow.model'`